source/Ship.cpp

```diff
@@ -2009,7 +2011,12 @@ void Ship::DoGeneration()
 	
 	heat -= heat * HeatDissipation();
 	if(heat > MaximumHeat())
-		isOverheated = true;
```

Comment:
The ship is overheated, so this should be set even for "cookable" ships.

Reply:
Done